src/components/Activity.tsx

```diff
@@ -0,0 +1,17 @@
+import { useState } from "react";
+import ActivityForm from "./ActivityForm";
+import ActivityDisplay from "./ActivityDisplay";
+import { ActivityFormData } from "@/interfaces/activity.interface";
+
+function Activity() {
+  const [data, setData] = useState<ActivityFormData | null>(null);
```

Comment:
Any reason to handle state here?
I only see the passing data between them
I can't see any operation here